Index: Smarty/templates/Header.tpl
===================================================================
--- Smarty/templates/Header.tpl	(revisión: 1679)
+++ Smarty/templates/Header.tpl	(copia de trabajo)
@@ -203,6 +203,7 @@
 </tr>
 </TABLE>
 <!-- - level 2 tabs starts-->
+{if $MODULE_NAME neq 'BIReporting'}
 <TABLE border=0 cellspacing=0 cellpadding=2 width=100% class="level2Bg" >
 <tr>
 	<td >
@@ -252,6 +253,7 @@
 </tr>
 </TABLE>		
 <!-- Level 2 tabs ends -->
+{/if}
 <div id="calculator_cont" style="position:absolute; z-index:10000" ></div>
 	{include file="Clock.tpl"}
 
